src/Compute/custom/NewAzVmss_SimpleParameterSet.cs

```diff
@@ -0,0 +1,373 @@
+            // config. We do not want to block that scenario in case the cmdlet failed mid operation and tthe user kicks it off again.
+            if (!loadBalancerNamePassedIn)
+            {
+                LoadBalancerStrategy.IgnorePreExistingConfigCheck = true;
+            }
+            else
+            {
+                LoadBalancerStrategy.IgnorePreExistingConfigCheck = false;
+            }
+
+            var result = await client.RunAsync(client.SubscriptionId, parameters, asyncCmdlet);
+
+
+            if (result != null)
+            {
+                var fqdn = PublicIPAddressStrategy.Fqdn(DomainNameLabel, Location);
+
+                // TODO: create custom type for FQDN
+                IVirtualMachineScaleSet psObject = result;
+
+                var port = "<port>";
+                var connectionString = parameters.ImageAndOsType.GetConnectionString(
+                    fqdn,
+                    Credential.UserName,
+                    port);
+                var range =
+                    FirstPortRangeStart.ToString() +
+                    ".." +
+                    (FirstPortRangeStart + InstanceCount * 2 - 1).ToString();
+
+                asyncCmdlet.WriteVerbose(
+                    Microsoft.Azure.PowerShell.Cmdlets.Compute.Resources.VmssUseConnectionString,
+                    connectionString);
+                asyncCmdlet.WriteVerbose(
+                    Microsoft.Azure.PowerShell.Cmdlets.Compute.Resources.VmssPortRange,
+                    port,
+                    range);
+                asyncCmdlet.WriteObject(psObject);
+            }
+        }
+
+        /// <summary>
+        /// Heres whats happening here :
+        /// If "SystemAssignedIdentity" and "UserAssignedIdentity" are both present we set the type of identity to be SystemAssignedUsrAssigned and set the user 
+        /// defined identity in the VMSS identity object.
+        /// If only "SystemAssignedIdentity" is present, we just set the type of the Identity to "SystemAssigned" and no identity ids are set as its created by Azure
+        /// If only "UserAssignedIdentity" is present, we set the type of the Identity to be "UserAssigned" and set the Identity in the VMSS identity object.
+        /// If neither is present, we return a null.
+        /// </summary>
+        /// <returns>Returning the Identity generated form the cmdlet parameters "SystemAssignedIdentity" and "UserAssignedIdentity"</returns>
+        private VirtualMachineScaleSetIdentity GetVmssIdentityFromArgs()
+        {
+            var identity = new VirtualMachineScaleSetIdentity();
+            if (this.IsParameterBound(c => c.SystemAssignedIdentity))
+            {
+                identity = new VirtualMachineScaleSetIdentity();
+                identity.Type = Microsoft.Azure.PowerShell.Cmdlets.Compute.Support.ResourceIdentityType.SystemAssigned;
+            }
+
+            // ToDO: Add support for User-Assigned identities
```

Comment:
@markcowl same comment here -- I think we should remove the `-UserAssignedIdentity` parameter if we're not going to do anything with it right now

Reply:
Leaving this for backward compatibility - it actually doesn't work correctly in the current cmdlet.